definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json

```diff
@@ -9,6 +9,7 @@
     "PreConditionState(s)": "IssuedToBailiff",
     "PostConditionState": "AosAwaiting",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/add-bailiff-return",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/purge-bailiff-data",
```

Comment:
why?

You know in `CallBackURLSubmittedEvent` you can't update case data.

Reply:
My mistake. I thought this would allow that